diagnostic.md

```diff
@@ -45,7 +59,33 @@ How would we reference each of the following categories of elements?
 -   All `p` elements whose parent elements are `div`s
 
 <!-- your answer starts here -->
+-   All elements that belong to the _class_ `big`
+```CSS
+.big {
+
+}
+```
+-   The element matching _ID_ `contentPane`
+```CSS
+#contentPane {
+
+}
+```
+
+-   Only those elements belonging to both the `important` and `red` _classes_
+```CSS
+.important.red {
 
+}
+```
+
+-   All `p` elements whose parent elements are `div`s
+We would need to assign a class to all of those p elements whose parent elements are divs. Let's say that class is called '.pdivs'. Then, we would reference those elements using the following code:
+```CSS
+.pdivs {
+
+}
+```
```

Comment:
You actually don't need to create a unique class. You can just use the div selector in this case.

`div > p { }`